Multisig/UI/WalletConnectV2/WalletConnectManager.swift

```diff
@@ -49,20 +49,20 @@ class WalletConnectManager {
                     NotificationCenter.default.post(name: .wcDidDisconnectSafeServer, object: self)
                 }
             }.store(in: &publishers)
-
-        Sign.instance.sessionProposalPublisher
+        
```

Comment:
Why we have this extra spaces ?

Reply:
These are inserted when executing `Editor -> Structure -> Re-Indent `in XCode